torch/csrc/jit/passes/xnnpack_rewrite.cpp

```diff
@@ -68,58 +72,62 @@ void insertXNNPACKConv2dOp(std::shared_ptr<Graph>& graph) {
         %r = aten::conv2d(%input, %weight, %bias, %stride, %padding, %dilation, %groups)
         return (%r) )";
 
-  std::string xnnpack_conv2d_pattern = R"(
+  std::string prepacked_ops_conv2d_pattern = R"(
     graph(%input, %weight, %bias, %stride:int[], %padding:int[], %dilation:int[], %groups:int):
-        %packed_weight_bias = _xnnpack::conv2d_prepack(%weight, %bias, %stride, %padding, %dilation, %groups)
-        %r = _xnnpack::conv2d_packed(%input, %packed_weight_bias)
+        %output_min_max : None = prim::Constant()
+        %packed_weight_bias = prepacked::conv2d_clamp_prepack(
+            %weight, %bias, %stride, %padding, %dilation, %groups,
+            %output_min_max, %output_min_max)
+        %r = prepacked::conv2d_clamp_run(%input, %packed_weight_bias)
         return (%r) )";
 
   SubgraphRewriter rewriter;
-  rewriter.RegisterRewritePattern(conv_2d_pattern, xnnpack_conv2d_pattern);
+  rewriter.RegisterRewritePattern(conv_2d_pattern, prepacked_ops_conv2d_pattern);
   rewriter.runOnGraph(graph);
 }
 
 } // namespace
 
-void insertXNNPACKOps(std::shared_ptr<Graph>& graph) {
+void insertPrePackedOps(std::shared_ptr<Graph>& graph) {
   ConstantPooling(graph);
   ConstantPropagation(graph);
-  insertXNNPACKLinearOp(graph);
-  insertXNNPACKConv2dOp(graph);
+  insertPrePackedLinearOp(graph);
+  insertPrePackedConv2dOp(graph);
 }
 
-void insertXNNPACKOps(script::Module& module) {
+void insertPrePackedOps(script::Module& module) {
   for (auto& method : module.get_methods()) {
     auto graph = method.graph();
-    insertXNNPACKOps(graph);
+    insertPrePackedOps(graph);
   }
   for (script::Module m : module.children()) {
-    insertXNNPACKOps(m);
+    insertPrePackedOps(m);
   }
 }
 
-void FoldXNNPACKPrePackingOps(script::Module& m) {
+void FoldPrePackingOps(script::Module& m) {
   PrePackingOpsFilterFn filter_fn = [](const Node* n) -> bool {
     return (
-        (n->kind() == Symbol::fromQualString("_xnnpack::linear_prepack")) ||
-        n->kind() == Symbol::fromQualString("_xnnpack::conv2d_prepack"));
+        (n->kind() ==
+         Symbol::fromQualString("prepacked::linear_clamp_prepack")) ||
+        n->kind() == Symbol::fromQualString("prepacked::conv2d_clamp_prepack"));
   };
-  FoldPrePackingOps(m, filter_fn, "xnnpack_prepack_folding");
+  PrePackingOpsFolder(m, filter_fn, "prepack_folding");
```

Comment:
I'm talking about the function name is changed from `FoldPrePackingOps` to `PrePackingOpsFolder`, which leads me to the PR, but it is not important. I'm just trying to understand what is the use case this function is targeting, is this only for non-quantized prepacking ops?

Reply:
Aah thanks for pointing out :). Shouldn't have changed that as part of this PR. But to answer your question, no. It should be generic as we had discussed originally when the PR for this pass was under review. It should still be a generally usable pass that is can be applied after freezing, regardless. However the naming either `FoldPrePackingOps` or `PrePackingOpsFolder` does suggest that it is for prepacking ops, including quantized. But I understand the concern, if that is the concern, that the naming may suggest that it is applied to ops that are in `prepacked` namespace even though that is not the case.

Reply:
Yeah the implementation is applicable to quantized ops as well. Do we want to do use this for quantized prepacking ops as well?

Reply:
I think that was the original plan, right? I dont think anything about that has changed, except naming. So I would say yes we should.

Reply:
I mean yeah, I can use `PrePackingOpsFolder` to implement a pass to fold quantized ops.
My question is should we merge that to the current new `FoldPrePackingOps` here, or is this still function still just for xnnpack?

Reply:
resolved in chat, this is prepack folding for fp32 ops.